From 721fc54c9411f11334a1912c5b274fbbda1e81f5 Mon Sep 17 00:00:00 2001
From: Markus Niebel <Markus.Niebel@ew.tq-group.com>
Date: Tue, 22 Nov 2022 08:01:43 +0100
Subject: [PATCH] iMX8ULP: remove unused targets

u-boot-atf.itb is not used but references unavailable ./mkimage_uboot.
u-boot-atf.bin is only a dependency for u-boot-atf.itb

Signed-off-by: Markus Niebel <Markus.Niebel@ew.tq-group.com>
---
 iMX8ULP/soc.mak | 11 -----------
 1 file changed, 11 deletions(-)

diff --git a/iMX8ULP/soc.mak b/iMX8ULP/soc.mak
index c1941a9..7fe10ff 100644
--- a/iMX8ULP/soc.mak
+++ b/iMX8ULP/soc.mak
@@ -38,17 +38,6 @@ u-boot-hash.bin: u-boot.bin
 	./$(MKIMG) -commit > head.hash
 	@cat u-boot.bin head.hash > u-boot-hash.bin
 
-u-boot-atf.bin: u-boot-hash.bin bl31.bin
-	@cp bl31.bin u-boot-atf.bin
-	@dd if=u-boot-hash.bin of=u-boot-atf.bin bs=1K seek=128
-
-u-boot-atf.itb: u-boot-hash.bin bl31.bin
-	./$(PAD_IMAGE) bl31.bin
-	./$(PAD_IMAGE) u-boot-hash.bin
-	TEE_LOAD_ADDR=$(TEE_LOAD_ADDR) ./mkimage_fit_atf.sh > u-boot.its;
-	./mkimage_uboot -E -p 0x3000 -f u-boot.its u-boot-atf.itb;
-	@rm -f u-boot.its
-
 u-boot-atf-container.img: bl31.bin u-boot-hash.bin
 	if [ -f tee.bin ]; then \
 		if [ $(shell echo $(ROLLBACK_INDEX_IN_CONTAINER)) ]; then \
-- 
2.25.1

